packages/contracts/src/governance/MainVotingPlugin.sol

```diff
@@ -199,6 +241,291 @@ contract MainVotingPlugin is Addresslist, MajorityVotingBase, IEditors, IMembers
+        });
+        proposal_.actions.push(_action);
+
+        proposalCreators[proposalId] = msg.sender;
+
+        emit ProposalCreated({
+            proposalId: proposalId,
+            creator: proposalCreators[proposalId],
+            metadata: _metadata,
+            startDate: _startDate,
+            endDate: proposal_.parameters.endDate,
+            actions: proposal_.actions,
+            allowFailureMap: 0
+        });
+    }
+
+    /// @notice Creates a proposal to remove an existing editor.
+    /// @param _metadata The metadata of the proposal.
+    /// @param _editor The address of the editor who may eveutnally be removed.
+    function proposeRemoveEditor(bytes calldata _metadata, address _editor) public onlyMembers {
+        if (!isEditor(_editor)) {
+            revert AlreadyNotAnEditor(_editor);
+        }
+        uint64 snapshotBlock;
+        unchecked {
+            snapshotBlock = block.number.toUint64() - 1; // The snapshot block must be mined already to protect the transaction against backrunning transactions causing census changes.
+        }
+        uint64 _startDate = block.timestamp.toUint64();
+
+        uint256 proposalId = _createProposalId();
+
+        // Store proposal related information
+        Proposal storage proposal_ = proposals[proposalId];
+
+        proposal_.parameters.startDate = _startDate;
+        proposal_.parameters.endDate = _startDate + duration();
+        proposal_.parameters.snapshotBlock = snapshotBlock;
+        proposal_.parameters.votingMode = votingMode();
+        proposal_.parameters.supportThreshold = supportThreshold();
+        IDAO.Action memory _action = IDAO.Action({
+            to: address(this),
+            value: 0,
+            data: abi.encodeCall(MainVotingPlugin.removeEditor, (_editor))
+        });
+        proposal_.actions.push(_action);
+
+        proposalCreators[proposalId] = msg.sender;
+
+        emit ProposalCreated({
+            proposalId: proposalId,
+            creator: proposalCreators[proposalId],
+            metadata: _metadata,
+            startDate: _startDate,
+            endDate: proposal_.parameters.endDate,
+            actions: proposal_.actions,
+            allowFailureMap: 0
+        });
+    }
+
+    /// @notice Determines whether at least one editor besides the creator has approved
+    /// @param _proposalId The ID of the proposal to check.
+    function isMinParticipationReached(uint256 _proposalId) public view override returns (bool) {
+        Proposal storage proposal_ = proposals[_proposalId];
+
+        if (proposal_.tally.yes == 0 && proposal_.tally.no == 0 && proposal_.tally.abstain == 0) {
+            return false;
+        }
+        // Just one voter?
+        else if (addresslistLengthAtBlock(proposal_.parameters.snapshotBlock) == 1) {
+            return true;
+        }
+
+        // More voters expected
+        return proposal_.nonCreatorsVoted;
+    }
```

Comment:
Does this handle the case where there is one voter but it's the creator (e.g., they're an editor)? In that case we want to require that _another_ editor votes on the proposal at minimum.

Reply:
Voters can only be editors. The case in which one voter is the creator will be handled by `addresslistLengthAtBlock()` first and then `nonCreatorsVoted`. 

- If there is only one potential voter, we return `true` because it's impossible that someone else can ever vote, and we don't want to brick the DAO.
- If there are more potential voters, then we check `nonCreatorsVoted` which returns `false` by default until another editor votes.

Isn't that the logic that you mention?

Reply:
Gotcha that makes sense 👍🏼